### PR TITLE
CR widget frontend (visual only)

### PR DESCRIPTION
**Pull request by hannahpurcell**
Frontend for the CR widget!

### CODE REVIEW

File: assets/css/pre_fare_v2.scss [hunk 1]
```diff
@@ -1,6 +1,7 @@
 @import url("https://rsms.me/inter/inter.css");
 
 @import "colors";
+@import "fonts_inter";
```
Comment (cmaddox5) [inline assets/css/pre_fare_v2.scss:4]: This shouldn't be necessary. We import it via the URL on line 1.

File: assets/src/components/v2/cr_departures.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                <td className="track-column">
+                  <div className="table-header__english">Track</div>
+                  <div className="table-header__spanish">Pista</div>
+                </td>
+              </tr>
+              { departures.slice(0, 3).map((departure, i) => {
+                  return (
+                    <tr key={i}>
+                      <td>{departure.arrow ? <Arrow direction={departure.arrow} className="departure__arrow-image" /> : "" }</td>
+                      {/* fix this so it flashes */}
```
Comment (cmaddox5) [inline assets/src/components/v2/cr_departures.tsx:46]: Still need this comment?

File: assets/src/components/v2/cr_departures.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                      <td className="headsign">{departure.headsign.headsign}</td>
+                      <td className="arrival">
+                        <div
+                          className={classWithModifier("departure-time", departure.time.type === "text" ? "animated" : "static")}
+                        >
+                          <BaseDepartureTime time={departure.time} hideAmPm/>
+                        </div>
+                      </td>
+                      <td className="track">{departure.track_number ? departure.track_number : ""}</td>
```
Comment (cmaddox5) [inline assets/src/components/v2/cr_departures.tsx:55]: I think you could simplify this a little.

```suggestion
                      <td className="track">{departure.track_number ?? ""}</td>
```

File: assets/src/components/v2/cr_departures.tsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+interface CRDeparturesProps {
+  departures: any,
+  destination: string,
+  time_to_destination: string
+}
+
+const CRDepartures: React.ComponentType<CRDeparturesProps> = (props) => {
+  const { departures, destination, time_to_destination } = props
+  console.log(props)
```
Comment (cmaddox5) [inline assets/src/components/v2/cr_departures.tsx:15]:
```suggestion
```

File: assets/css/pre_fare_v2.scss [hunk 5]
```diff
@@ -43,6 +45,10 @@
 
 @import "v2/pre_fare/simulation";
 
+// Don't import from solari... put in common place!
```
Comment (cmaddox5) [inline assets/css/pre_fare_v2.scss:48]: I think we all three ended up using the Solari arrow, so this comment will be super helpful.

**Review by cmaddox5 — CHANGES_REQUESTED** [review]
Nailed it! Just have a couple of suggestions/questions.

File: assets/src/components/v2/cr_departures.tsx (hunk 3 above)
Comment (hannahpurcell) [reply assets/src/components/v2/cr_departures.tsx:55]: WHOA!! I didn't know of this syntax. That's so exciting

File: assets/css/pre_fare_v2.scss (hunk 1 above)
Comment (hannahpurcell) [reply assets/css/pre_fare_v2.scss:4]: Unfortunately, the `solari/departure_time` file has special mixins that are in the fonts file 🙄

File: assets/src/components/admin/admin_tables.tsx [hunk 6]
```diff
@@ -611,14 +611,31 @@ const blueBikesColumn = {
...
   return (
     <AdminTable
-      columns={[...v2Columns, lineMapColumn, elevatorStatusColumn, reconstructedAlertWidgetColumn, contentSummaryColumn, blueBikesColumn]}
+      columns={[
+        ...v2Columns,
+        lineMapColumn,
+        elevatorStatusColumn,
+        reconstructedAlertWidgetColumn,
+        contentSummaryColumn,
+        blueBikesColumn,
```
Comment (cmaddox5) [inline assets/src/components/admin/admin_tables.tsx:636]: Definitely doesn't need to block you right now, but a column will need to be added for this widget.

File: assets/static/images/free-green.svg [hunk 7]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (cmaddox5) [inline assets/static/images/free-green.svg:1]: Should be able to delete this file now that the `Free` component is merged in.

**Review by cmaddox5 — APPROVED** [review]
Looks great! A couple of comments, but nothing urgent.

**Pull request merged**